### ISSUE TITLE
Jittering while spectating a game with behindview 0

### ISSUE THREAD

**Issue by garry-ut99** [OldUnreal/UnrealTournamentPatches#1267]
While spectating a game, after typing `behindview 0` to spectate in 1st person, jittering appears.
After switchting back to `behindview 1`, the jittering dissappears, so only `behindview 0` is affected.
It happens while spectating an online game or a demoplay, but not when spectating a local off-line game with bots.
Using [SPECFIX](http://fpsnetwork.com/downloads/index.php?path=ut99%2Fappz+%26+utilities%2Fspecfix/) like advised [IN THIS THREAD](https://eatsleeput.com/d/307-cheater-fx-e3xecutioner) fixes the issue when spectating online games, but doesn't fix when playing demos. Other players already confirmed the issue, and they complain about jittering: [LINK 1](https://eatsleeput.com/d/297-cheater-ruffled-beaver) + [LINK 2](https://eatsleeput.com/d/307-cheater-fx-e3xecutioner )

**Comment by SeriousBuggie** [comment]
I suspect this jitter caused by replication data, which come from net. Update player rotation. Possible spec fix make smooth of it.

**Comment by stijn-volckaert** [comment]
Indeed. This is a fundamental limitation of the UE1 netcode. Servers try to save bandwidth by compressing some of the data they send to players. This compression costs us some precision in things like (camera/player) rotation updates
